GrowthViz-adults.py

```diff
@@ -1,21 +1,23 @@
 #!/usr/bin/env python
 # coding: utf-8
 
-# # GrowthViz
+# # GrowthViz - Adults
 # 
 # ## Background
 # 
 # As stated in [Automated identification of implausible values in growth data from pediatric electronic health records](https://academic.oup.com/jamia/article/24/6/1080/3767271):
 # 
 # > In pediatrics, evaluation of growth is fundamental, and many pediatric research studies include some aspect of growth as an outcome or other variable. The clinical growth measurements obtained in day-to-day care are susceptible to error beyond the imprecision inherent in any anthropometric measurement. Some errors result from minor problems with measurement technique. While these errors can be important in certain analyses, they are often small and generally impossible to detect after measurements are recorded. Larger measurement technique errors can result in values that are biologically implausible and can cause problems for many analyses. 
 # 
+# The `growthcleanr` package has been expanded to identify implausible height and weight observations among adult subjects, up to age 65. This notebook, like the pediatrics-focused `GrowthViz-pediatrics.ipynb`, supports review of the output `growthcleanr` when working with data from adult subjects.
+# 
 # ## Growthcleanr
 # 
-# [growthcleanr](https://github.com/carriedaymont/growthcleanr) is an automated method for cleaning longitudinal pediatric growth data from EHRs. It is available as open source software.
+# [growthcleanr](https://github.com/carriedaymont/growthcleanr) is an automated method for identifying implausible values in adult anthropometric data from EHRs. It is available as open source software.
```

Comment:
Maybe we should maintain the same description in each notebook of what growthcleanr does - just to make it clear that it's not focused only on one or the other? So basically combining the information from these two lines instead of removing the reference to pediatrics?

Reply:
Yes, that's exactly what I was thinking in that comment above. I can do this now - wanted to merge in the changes from #14 first to resolve any conflicts.

Reply:
Updated and slightly reorganized the intro text in both notebooks to align better. How's it read?